docs/dev_notes.md

```diff
@@ -13,26 +13,28 @@ The build system has been recently revamped. Please see details in the file
 
 ### Unit Testing (rspec, puppet-lint, etc)
 
-| OS           | Ruby  | Puppet |
-|--------------|-------|--------|
-| RHEL 6       | 1.8.7 | 3.8.7  |
-| RHEL 7       | 2.0.0 | 3.8.7  |
-| Ubuntu 14.04 | 1.9.3 | 3.8.7  |
-| Ubuntu 16.06 | 2.3.1 | 3.8.5  |
+| OS            | Ruby  | Puppet |
+|-------------- |-------|--------|
+| Ubuntu 20.04 | 2.5   | 6      |
+| Ubuntu 20.04 | 2.7   | 7      |
 
 
 ### Integration Testing (test-kitchen)
 
-Note: "Base" specs are for the Travis CI container that test-kitchen is
+Note: "Base" specs are for the Github Action container that test-kitchen is
 being run on. All other columns are details about the guest OS that is 
 created by test-kitchen that puppet-st2 is run against.
 
-| Base OS (travis) | Base Ruby | Guest OS     | Guest Ruby | Guest Puppet |
-|------------------|-----------|--------------|------------|--------------|
-| Ubuntu 14.04     | 2.4       | RHEL 6       | 1.8.7      | 3.8.7        |
-| Ubuntu 14.04     | 2.4       | RHEL 7       | 2.0.0      | 3.8.7        |
-| Ubuntu 14.04     | 2.4       | Ubuntu 14.04 | 1.9.3      | 3.8.7        |
-| Ubuntu 14.04     | 2.4       | Ubuntu 16.06 | 2.3.1      | 3.8.5        |
+| Base OS (Github) | Base Ruby | Guest OS     | Guest Ruby | Guest Puppet | Guest Python |
+|------------------|-----------|--------------|------------|--------------|--------------|
+| Ubuntu 20.04     | 2.7       | RHEL 7       | 2.7        | 6            | system       |
+| Ubuntu 20.04     | 2.7       | RHEL 7       | 2.7        | 7            | system       |
+| Ubuntu 20.04     | 2.7       | RHEL 7       | 2.7        | 7            | 3.6          |
+| Ubuntu 20.04     | 2.7       | Ubuntu 18.04 | 2.7        | 6            | system       |
+| Ubuntu 20.04     | 2.7       | Ubuntu 18.04 | 2.7        | 7            | system       |
+| Ubuntu 20.04     | 2.7       | Ubuntu 18.04 | 2.7        | 7            | 3.6          |
```

Comment:
As Guest Python for 18.04 is 3.6, why two entries? Might relate to the other points in earlier files about that python36 run.

Reply:
Yup, same as the old points. Will get rid of this reference too.